_posts/2018-03-05-crosswords.html

```diff
@@ -46,6 +49,9 @@ <h3>A cross-platform feature: the list view</h3>
 	<figcaption>The grid and list views on mobile</figcaption>
 </figure>
 
+{% include youtube_video.html src="https://www.youtube.com/embed/QzqLHXBsO3w" %}
```

Comment:
The whole point of mobile is to *not* interact with the grid. Can we redo a video where the interaction is in the clue?

Reply:
Image is probably superfluous too